installer/scripts/omsadmin.sh

```diff
@@ -30,6 +30,7 @@ CONF_OMSAGENT=$CONF_DIR/omsagent.conf
 
 # Omsagent proxy configuration
 CONF_PROXY=$ETC_DIR/proxy.conf
+OLD_CONF_PROXY=/etc/opt/microsoft/omsagent/conf/proxy.conf
 
```

Comment:
Changed to use $DF_CONF_DIR/proxy.conf instead since that is the convention [here](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/scripts/omsadmin.sh#L652)